frontend/src/metabase/dashboard/actions.js

```diff
@@ -1127,7 +1127,8 @@ export const fetchDashboardParameterValuesWithCache = createThunkAction(
 
       return {
         cacheKey,
-        results: results.map(result => [].concat(result)),
+        results: values.map(value => [].concat(value)),
+        has_more_values: query ? true : has_more_values,
```

Comment:
The `DashboardApi.parameterSearch` endpoint returns a `has_more_values` prop as well, but the components aren't set up to use it, so we need to always pass `true` in these scenarios.